src/views/pages/not-found.ejs

```diff
@@ -0,0 +1,19 @@
+<%- include("../includes/head", { title: "404 - Сторінку не знайдено"}) %>
+    <link rel="stylesheet" href="/css/pages/not-found.css">
+    <link rel="stylesheet" href="/css/components/link.css">
+    </head>
+    <body>
+        <main class="main">
+            <div class="error-container">
+                <h1 class="error-code">404</h1>
+                <p class="error-description">Сторінку не знайдено</p>
+                <%- include("../components/link.ejs", {
+                    href: "/",
```

Comment:
```suggestion
                    href: homePath,
```

pass it as a parameter, using PagesPath enum, checkout how we pass route link for sign in and sign up when render home page